test/it/fixtures/test.suite.ts

```diff
@@ -0,0 +1,344 @@
+//
+// @suite class Times {
+//     @test @slow(10) public "when fast is normal"(done) {
+//         setTimeout(done, 0);
+//     }
+//     @test @slow(15) public "when average is yellow-ish"(done) {
+//         setTimeout(done, 10);
+//     }
+//     @test @slow(15) public "when slow is red-ish"(done) {
+//         setTimeout(done, 20);
+//     }
+//     @test @timeout(10) public "when faster than timeout passes"(done) {
+//         setTimeout(done, 0);
+//     }
+//     @test @timeout(10) public "when slower than timeout fails"(done) {
+//         setTimeout(done, 20);
+//     }
+// }
+//
+// //   Times
+// //     √ when fast is normal
+// //     √ when average is yellow-ish (10ms)
+// //     √ when slow is red-ish (20ms)
+// //     √ when faster than timeout passes
+// //     6) when slower than timeout fails
+//
+// @suite class ExecutionControl {
+//     @skip @test public "this won't run"() {
+//     }
+//
+//     @test public "this however will"() {
+//     }
+//
+//     // @only
+//     @test public "add @only to run just this test"() {
+//     }
+// }
+//
+// //   ExecutionControl
+// //     - this won't run
+// //     √ this however will
+// //     √ add @only to run just this test
+//
+// class ServerTests {
+//     public connect() {
+//         console.log("      connect(" + ServerTests.connection + ")");
+//     }
+//     public disconnect() {
+//         console.log("      disconnect(" + ServerTests.connection + ")");
+//     }
+//
+//     public static connection: string;
+//     public static connectionId: number = 0;
+//
+//     public static before() {
+//         ServerTests.connection = "shader connection " + ++ServerTests.connectionId;
+//         console.log("    boot up server.");
+//     }
+//
+//     public static after() {
+//         ServerTests.connection = undefined;
+//         console.log("    tear down server.");
+//     }
+// }
+//
+// @suite class MobileClient extends ServerTests {
+//     @test public "client can connect"() { this.connect(); }
+//     @test public "client can disconnect"() { this.disconnect(); }
+// }
+//
+// @suite class WebClient extends ServerTests {
+//     @test public "web can connect"() { this.connect(); }
+//     @test public "web can disconnect"() { this.disconnect(); }
+// }
+//
+// //   MobileClient
+// //   boot up server.
+// //     connect(shader connection 1)
+// //     √ client can connect
+// //     disconnect(shader connection 1)
+// //     √ client can disconnect
+// //   tear down server.
+//
+// //   WebClient
+// //   boot up server.
+// //     connect(shader connection 2)
+// //     √ web can connect
+// //     disconnect(shader connection 2)
+// //     √ web can disconnect
+// //   tear down server.
+//
+// @suite @timeout(10) class OverallSlow {
+//     @test public "first fast"(done) {
+//         setTimeout(done, 1);
+//     }
+//     @test public "second slow"(done) {
+//         setTimeout(done, 20);
+//     }
+//     @test public "third fast"(done) {
```

Comment:
I wonder if I let all this commented tests slip in a PR

Reply:
reactivate?

Reply:
reactivated - guess what, the coverage just went up 👍

Reply:
🦅 eye... great!